### ISSUE TITLE
Contribute a docset to Dash

### ISSUE THREAD

**Issue by underyx** [coala-analyzer/documentation#5]
Dash is a documentation searcher/browser app for macOS (it also has independent reimplementations for other platforms).

https://github.com/Kapeli/Dash-User-Contributions#contribute-a-new-docset

**Comment by sils1297** [comment]
Can you open an issue at dash?

**Comment by underyx** [comment]
I'd rather not, the bears don't even have HTML doc generation set up yet I believe, and I really doubt that anyone would just pick up requests from the Dash repo.

I wanted to open this issue in some meta issue tracker for coala but this is the best repo I've found. Getting coala on devdocs.io would be an example of an issue that would be related and nice to track in the same place as this one. If I were to open one for devdocs.io it'd be stuck for quite a while since they require >3k GitHub stars.

**Comment by jayvdb** [comment]
@Monal5031 , can you investigate what is involved.  Is it a possible GCI task?  Do we have all the necessary content written?  Or is it too big?

**Comment by Monal5031** [comment]
Yeah, this can be a GCI task but is it okay to have a PR over dash repo and count it as a GCI task? , also it requires info on how to generate the docs.

A sample is given there on how to contribute the doc https://github.com/Kapeli/Dash-User-Contributions/tree/master/Sample_Docset
According to the sample doc I think we have all the necessary things
- [x] Author
- [x] Instruction to generate the doc
P.S.: We'd still have to think about having a GCI PR in another repo.
